src/app/features/stacks-transaction-request/minimal-error-message.tsx

```diff
@@ -37,7 +37,7 @@ function MinimalErrorMessageSuspense(props: HstackProps) {
       // #4476 TODO change this colour
       bg="#FCEEED"
       borderRadius="md"
-      color="error.label"
+      color="red.action-primary-default"
```

Comment:
I just saw this `TODO` above now from the stacks UI removal days (oh we had fun)

I think we can do it now and `bg="#FCEEED"` can become `bg="ink.background-primary"